src/components/ReviewAnswers.js

```diff
@@ -24,20 +24,21 @@ const ReviewAnswers = ({
         Your score is: {score} / {numOfQuestions}
       </p>
       {questions.map(question => (
-        <Card className="mb-4">
+        <Card key={question.id} className="mb-4">
           <Card.Body>
             <Card.Title>{question.text}</Card.Title>
-            <Card.Text>
+            <div>
```

Comment:
I had a look and can see that `Card.Text` gets rendered as a `p`, and MDN says `ul`s should not go within `p`s so good job changing this. However, here the `div` is not doing anything (whereas before `Card.Text` could've had a specific style associated with it which would've made it necessary) so we can remove the `div`

Reply:
I will fix it, thanks 😊

Reply:
yes, about that error, Vicky mentioned to me to fix.